cla_backend/settings/base.py

```diff
@@ -342,6 +345,19 @@ def traces_sampler(sampling_context):
 SECURE_CONTENT_TYPE_NOSNIFF = True
 SESSION_COOKIE_SAMESITE = "strict"
 
+CSP_DEFAULT_SRC = ["'self'"]
+CSP_SCRIPT_SRC = ["'self'", "'unsafe-inline'", "cloud-platform-6cf3132ef8fce52bb371b1d02f40c36d.s3.amazonaws.com"]
```

Comment:
Each environment has it's own s3 bucket. The hostname of the s3 bucket is determined later down in line 356 so we don't need to hard code one here

Reply:
Removed :D